connector-runtime/connector-runtime-spring/src/main/java/io/camunda/connector/runtime/inbound/controller/InboundConnectorRestController.java

```diff
@@ -88,16 +92,33 @@ private List<ActiveInboundConnectorResponse> getActiveInboundConnectors(
         .collect(Collectors.toList());
   }
 
-  private Map<String, Object> getData(ActiveExecutableResponse connector) {
-    Map<String, Object> data = Map.of();
+  /**
+   * Be aware that this API is used by c4-connectors to get the active inbound connectors grouped by
+   * connector connectorId. Changing the response format will break the c4-connectors, so make sure
+   * to update the c4-connectors as well.
+   */
+  private List<ConnectorInstances> getConnectorsInstances(
+      String bpmnProcessId, String elementId, String type) {
+    var activeInboundConnectors = getActiveInboundConnectors(bpmnProcessId, elementId, type, null);
+    return activeInboundConnectors.stream()
+        .collect(Collectors.groupingBy(ActiveInboundConnectorResponse::type, Collectors.toList()))
+        .entrySet()
+        .stream()
+        .map(e -> new ConnectorInstances(e.getKey(), e.getValue()))
+        .collect(Collectors.toList());
+  }
+
+  private Map<String, String> getData(ActiveExecutableResponse connector) {
+    Map<String, String> data =
+        new HashMap<>(connector.elements().getFirst().connectorLevelProperties());
```

Comment:
As discussed in our chat we need to add all connector elements to make sure we have all individual properties available in the UI.